src/opm/parser/eclipse/share/keywords/000_Eclipse100/N/NETBALAN

```diff
@@ -15,7 +15,7 @@
     {
       "name": "PRESSURE_CONVERGENCE_LIMIT",
       "value_type": "DOUBLE",
-      "default": 1e-01,
+      "default": 1e+04,
```

Comment:
Ehhh - I don't really agree here; I read `0.1 Bars` from the manual. Observe that the default values given in the Json specification should be in `SI` units but rather in `METRIC` units - and in the case of pressure those two differ?

Reply:
This property is a pressure and when I use the unit-conversion in CreateDoubHead, the unit used in flow is in Pa, else I would not get the correct value output to the restart file?

Reply:
Well - then there is a bug somewhere else - I have a hunch.

Reply:
Fix here: https://github.com/OPM/opm-common/pull/2870

Reply:
I have now corrected this change back to the original value since the master code now works as intended.